readme.md

```diff
@@ -153,7 +153,7 @@ When running under Apache MPM, only `prefork` mode is supported, since `dl()` ca
 
 #### Refresh Internal Data
 
-To reload the data file, *refreshData()* needs to be called as described in this [example](http://51degrees.com/documentation/_examples__device_detection__data_file_updates__manual.html) . Thus, unless a reload is performed in the main process and all child processes, the data will not be updated. There is not a proven solution to do so yet, so we recommend a full server restart to be performed instead.
+To reload the data file, *refreshData()* needs to be called as described in this [example](https://51degrees.com/documentation/_examples__device_detection__data_file_updates__manual.html) . Thus, unless a reload is performed in the main process and all child processes, the data will not be updated. There is not a proven solution to do so yet, so we recommend a full server restart to be performed instead.
```

Comment:
I notice this is a change to https, but another PR has made a change the other way around. Do we know which we are standardizing on?

Reply:
this was requested by me! mainly because i noticed it ages ago and it was bugging me. not sure on what we are saying is standard, but from my marketing point of view, we shouldnt be linking to any http pages, only https